src/app/calendar/page.tsx

```diff
@@ -1,14 +1,50 @@
 import { Metadata } from "next";
 import React from "react";
 import ParcelCalendar from "@/app/calendar/ParcelCalendar";
-import { getParcelsWithCollectionDate } from "@/app/calendar/parcelCalendarFunctions";
+import { ParcelWithClientName } from "@/app/calendar/parcelCalendarFunctions";
+import { Schema } from "@/database_utils";
+import supabase from "@/supabaseServer";
+export const revalidate = 0;
```

Comment:
new line between imports and 'actual code' please :)

Reply:
Yes sorry, webstorm interface makes me thought there is a new line TT